youtube_dl/extractor/la7.py

```diff
@@ -10,17 +10,17 @@
 
 class LA7IE(InfoExtractor):
     IE_NAME = 'la7.it'
-    _VALID_URL = r'''(?x)(https?://)?(?:
-        (?:www\.)?la7\.it/([^/]+)/(?:rivedila7|video)/|
-        tg\.la7\.it/repliche-tgla7\?id=
+    _VALID_URL = r'''(?x)(https?://)(?:www\.)?la7\.it/(?:
+        [^/]+/(?:rivedila7|video)/|
+        embedded/la7[^/]*(?:&|\?)content=
     )(?P<id>.+)'''
 
     _TESTS = [{
         # 'src' is a plain URL
         'url': 'http://www.la7.it/crozza/video/inccool8-02-10-2015-163722',
         'md5': '8b613ffc0c4bf9b9e377169fc19c214c',
         'info_dict': {
-            'id': 'inccool8-02-10-2015-163722',
+            'id': '0_42j6wd36',
```

Comment:
It's better to keep the original ID for existing patterns, or --download-archive will be broken.

Reply:
I did not change the id extraction, it changed in the Kaltura extractor.